kodilla-stream/build/reports/tests/test/classes/com.kodilla.stream.beautifier.BeautifierTestSuite.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
```

Comment:
specjalnie to dorzuciłeś?

Reply:
Nie.

Reply:
Oki :) Mogę Ci przy następnym spotkaniu pokazać jak usunąć dodać do gitignore`a pliki jak już je zacommitujesz przez przypadek i je później usunąć :)